app/boards/arm/corneish_zen/corneish_zen_v2_right_defconfig

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2022 Darryl deHaan
+# SPDX-License-Identifier: MIT
+#
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_CORNEISH_ZEN_V2_RIGHT=y
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SLEEP=y
+CONFIG_ZMK_DISPLAY=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable SPI
+CONFIG_SPI=y
+CONFIG_SPI_NRFX=y
+
+# Enable writing to flash
+CONFIG_USE_DT_CODE_PARTITION=y
+CONFIG_BUILD_OUTPUT_UF2=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+CONFIG_NVS=y
+CONFIG_SETTINGS_NVS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+
+# Enable 32kHz crystal
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL=y
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_30PPM=y
+
+# enable display drivers
+CONFIG_ZMK_DISPLAY_WORK_QUEUE_DEDICATED=y
+CONFIG_ZMK_DISPLAY_DEDICATED_THREAD_STACK_SIZE=2048
+CONFIG_SSD1306=n
+CONFIG_IL0323=y
+CONFIG_LVGL_HOR_RES_MAX=80
+CONFIG_LVGL_VER_RES_MAX=128
+CONFIG_LVGL_BITS_PER_PIXEL=1
+CONFIG_LVGL_COLOR_DEPTH_1=y
+CONFIG_LVGL_DPI=145
+CONFIG_LVGL_VDB_SIZE=100
+CONFIG_LVGL_USE_THEME_MONO=y
+CONFIG_LVGL_THEME_DEFAULT_COLOR_PRIMARY_RED=n
+CONFIG_LVGL_THEME_DEFAULT_COLOR_SECONDARY_RED=n
+CONFIG_LVGL_THEME_DEFAULT_COLOR_PRIMARY_BLACK=y
+CONFIG_LVGL_THEME_DEFAULT_COLOR_SECONDARY_WHITE=y
+CONFIG_LVGL_THEME_DEFAULT_FONT_SMALL_MONTSERRAT_16=y
+CONFIG_LVGL_THEME_DEFAULT_FONT_NORMAL_MONTSERRAT_26=y
+
+# custom status screens
+CONFIG_ZMK_DISPLAY_STATUS_SCREEN_CUSTOM=y
+CONFIG_ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN=n
+CONFIG_CUSTOM_WIDGET_BATTERY_STATUS=y
+CONFIG_ZMK_WIDGET_BATTERY_STATUS=n
+CONFIG_CUSTOM_WIDGET_PERIPHERAL_STATUS=y
+CONFIG_ZMK_WIDGET_PERIPHERAL_STATUS=n
+
+# Turn on logging, and set ZMK logging to debug output
+#CONFIG_LOG=y
+#CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=8000
+#CONFIG_ZMK_USB_LOGGING=y
+#CONFIG_ZMK_LOG_LEVEL_DBG=y
+#CONFIG_LOG_BUFFER_SIZE=20000
+#CONFIG_LOG_STRDUP_BUF_COUNT=60
+#CONFIG_I2C_LOG_LEVEL_DBG=y
+#CONFIG_SPI_LOG_LEVEL_DBG=y
+#CONFIG_DISPLAY_LOG_LEVEL_DBG=y
+#CONFIG_LVGL_LOG_LEVEL_DBG=y
+#CONFIG_LVGL_USE_DEBUG=y
+#CONFIG_SENSOR_LOG_LEVEL_DBG=y
+
+# Turn on USB CDC ACM device
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_CDC_ACM_RINGBUF_SIZE=1024
```

Comment:
These usually aren't enabled by default, why here?

Reply:
I was likely trying to solve an issue by trying everything I could think of. Will delete